src/domain/valueObject/helper/interfaceToUint16.go

```diff
@@ -0,0 +1,10 @@
+package voHelper
+
+func InterfaceToUint16(input interface{}) (output uint16, err error) {
```

Comment:
I take this will never fly for 100 bucks, hehe. Please follow uint8/32 approach checking if the value sent exceeds the maximum value accepted by the final type.

Reply:
Changed to **Control**'s format.